File: src/main/java/com/github/wgzhao/dbquery/entities/QueryLog.java
package com.github.wgzhao.dbquery.entities;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Table(name="query_logs")
@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
public class QueryLog
        extends BaseEntity {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private long id;

    private String appId;

    private String selectId;

    private String querySql;

}

File: src/main/kotlin/com/github/wgzhao/dbquery/entities/QueryLog.kt
package com.github.wgzhao.dbquery.entities

import jakarta.persistence.Id
import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Table
import java.sql.Timestamp

@Entity
@Table(name = "query_logs")
data class QueryLog(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long = 0,
    var appId: String,
    var selectId: String,
    var querySql: String,
    val createdAt: Timestamp = Timestamp(System.currentTimeMillis()),
    val updatedAt: Timestamp = Timestamp(System.currentTimeMillis())
)
